src/routes/+page.svelte

```diff
@@ -1,3 +1,32 @@
-<h1>Welcome to your library project</h1>
-<p>Create your package using @sveltejs/package and preview/showcase your work with SvelteKit</p>
-<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
+<script lang="ts">
+	import { flip, offset } from '@floating-ui/dom';
+	import { useFloating } from '$lib/hooks/use-floating.svelte.js';
+
+	const floating = useFloating({
```

Comment:
Were you still leaning towards destructuring here or was there issue with this approach?

Reply:
We cannot destructure do the limitiation of Svelte runes. It would lose reactivity.